packages/next/src/server/async-storage/with-request-store.ts

```diff
@@ -20,7 +20,7 @@ import {
 import { ResponseCookies, RequestCookies } from '../web/spec-extension/cookies'
 import { DraftModeProvider } from './draft-mode-provider'
 import { splitCookiesString } from '../web/utils'
-import { createAfterContext, type AfterContext } from '../after/after-context'
+import { AfterContextImpl, type AfterContext } from '../after/after-context'
```

Comment:
i don't feel strongly about this, but for me it looks a bit weird/leaky to import "Impl" things across modules. i liked `createAfterContext` because it hid this and only exposed the interface